service-jehlomat/src/main/kotlin/api/SyringeApi.kt

```diff
@@ -34,6 +34,42 @@ fun Route.syringeApi(database: DatabaseService, mailer: MailerService): Route {
             call.respond(HttpStatusCode.OK, SyringeFilterResponse(filteredSyringes, pageInfo))
         }
 
+        authenticate(JWT_CONFIG_NAME) {
+
+            post("export") {
+                val loggedInUser = jwtManager.getLoggedInUser(call, database)
+                val user = database.selectUserById(loggedInUser.id)
+                val organization = user?.let { it1 -> database.selectOrganizationById(it1.organizationId) }
+                val roles = PermissionService.determineRoles(loggedInUser, organization)
+                if (!roles.contains(Role.OrgAdmin) && !roles.contains(Role.SuperAdmin)) {
+                    call.respond(HttpStatusCode.Forbidden, "Only admin or superadmin can export database")
+                    return@post
+                }
+
+                val request = call.receive<SyringeFilter>()
+
+                val organizationId = if (roles.contains(Role.SuperAdmin)) {
+                    null
+                } else {
+                    database.selectUserById(loggedInUser.id)?.organizationId
```

Comment:
You could use here just loggedInUser.organizationId

Reply:
c9ac85de636fc91d199464d219f9de030c7cdf37